demisto_sdk/commands/download/downloader.py

```diff
@@ -237,42 +247,41 @@ def verify_flags(self) -> bool:
             output_flag, input_flag = True, True
             if not self.output_pack_path:
                 output_flag = False
-                logger.info("[red]Error: Missing option '-o' / '--output'.[/red]")
+                logger.error("Error: Missing required parameter '-o' / '--output'.")
             if not self.input_files:
                 if not self.all_custom_content and not self.regex:
                     input_flag = False
-                    logger.info("[red]Error: Missing option '-i' / '--input'.[/red]")
+                    logger.error("Error: Missing required parameter '-i' / '--input'.")
             if not input_flag or not output_flag:
                 is_valid = False
 
         if self.download_system_item and not self.system_item_type:
-            logger.info(
-                "[red]Error: Missing option '-it' / '--item-type', "
-                "you should specify the system item type to download.[/red]"
+            logger.error(
+                "Error: Missing required parameter '-it' / '--item-type'."
```

Comment:
aren't all these validated by `click`? 🤔

Reply:
These flags are checked under specific conditions (ex: `if self.download_system_item and not self.system_item_type`, so I think not.

But I'll leave that for the larger follow-up refactor.

Reply:
Cool, please write it down.

Reply:
I've opened [CIAC-7421](https://jira-hq.paloaltonetworks.local/browse/CIAC-7421)
Feel free to add notes / tasks there as well for things that should be changed / updated outside the scope of this PR.